.github/actions/build/action.yml

```diff
@@ -14,9 +14,6 @@ inputs:
   GROWTHBOOK_CLIENT_KEY:
     description: Growthbook client key for analytics
     required: false
-  GROWTHBOOK_DECRYPTION_KEY:
-    description: Growthbook decryption key for analytics
-    required: false
```

Comment:
GROWTHBOOK_DECRYPTION_KEY is not needed in the new analytics package update, thus removing it